src/fsharp/FSharp.Core/list.fs

```diff
@@ -163,7 +163,7 @@ namespace Microsoft.FSharp.Collections
         let choose chooser list = Microsoft.FSharp.Primitives.Basics.List.choose chooser list
 
         [<CompiledName("SplitAt")>]
-        let splitAt index (list:'T list) = Microsoft.FSharp.Primitives.Basics.List.splitAt index list
+        let splitAt count (list:'T list) = Microsoft.FSharp.Primitives.Basics.List.splitAt count list
```

Comment:
Wouldn't renaming this parameter be a breaking change for anyone consuming this API from, e.g., C#, if they were using named arguments? The following would break, for example, because the first parameter would no longer be named `index`:
```csharp
var list = ListModule.OfArray(new[] { 1, 2, 3, 4 });
var (a, b) = ListModule.SplitAt(index: 2, list: list);
```

Reply:
yes, it is break, but splitAt is newer. correct it as early as possible. index less 1 than count, it is mistake.